Tests/SentryTests/SentryTransportInitializerTests.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SentryTransportInitializerTests.swift
+//  SentryTests
+//
+//  Created by Philipp Hofmann on 08.04.20.
+//  Copyright © 2020 Sentry. All rights reserved.
+//
```

Comment:
I was hoping we drop these file headers (which IMO are just noise). 
If you agree we can could stop adding them and later use a tool (or write a script) to remove them all from the code base.

Reasoning: The date and author are in git. File name doesn't help (IDE shows it). File name doesn't get applied by refactoring often and gets out dated. 8 lines of useless info before we can read code.

Reply:
I agree, I also don't like them. 👏